dev-libs/inih/inih-r56.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="Simple .INI file parser in C, good for embedded systems"
+DESCRIPTION="a simple .INI file parser written in C. It's only a couple of pages of code, and it was designed to be small and simple"
```

Comment:
Forgot to mention, wrap the line at the end with a max of 100 characters (same for tio recipe): https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#description